parquet-hadoop/src/main/java/org/apache/parquet/hadoop/InternalParquetRecordReader.java

```diff
@@ -111,18 +111,18 @@ private void checkRead() throws IOException {
     if (current == totalCountLoadedSoFar) {
       if (current != 0) {
         totalTimeSpentProcessingRecords += (System.currentTimeMillis() - startedAssemblingCurrentBlockAt);
-        if (LOG.isInfoEnabled()) {
-            LOG.info("Assembled and processed " + totalCountLoadedSoFar + " records from " + columnCount + " columns in " + totalTimeSpentProcessingRecords + " ms: "+((float)totalCountLoadedSoFar / totalTimeSpentProcessingRecords) + " rec/ms, " + ((float)totalCountLoadedSoFar * columnCount / totalTimeSpentProcessingRecords) + " cell/ms");
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("Assembled and processed " + totalCountLoadedSoFar + " records from " + columnCount + " columns in " + totalTimeSpentProcessingRecords + " ms: "+((float)totalCountLoadedSoFar / totalTimeSpentProcessingRecords) + " rec/ms, " + ((float)totalCountLoadedSoFar * columnCount / totalTimeSpentProcessingRecords) + " cell/ms");
```

Comment:
```suggestion
            LOG.debug("Assembled and processed {} records from {} columns in {} ms: {} rec/ms, {} cell/ms", totalCountLoadedSoFar, columnCount, totalTimeSpentProcessingRecords, ((float)totalCountLoadedSoFar / totalTimeSpentProcessingRecords), ((float)totalCountLoadedSoFar * columnCount / totalTimeSpentProcessingRecords));
```

Reply:
I agree with @Fokko. Could you update this?